src/features/ExploreView/ExploreMenu.js

```diff
@@ -154,6 +154,7 @@ const ExploreMenu = ({ navigation }) => {
               individualBills: 0,
               date: Timestamp.fromDate(new Date()),
               role: 'Host',
+              restaurantName: restaurantName,
```

Comment:
lol sorry for jumping back and forth on comments - just realized we need this line for guests as well! probably on line 105 in `GuestQR.js`. Thanks!